scripts/azure_new_vm_size_provision/watcher-new-azure-vm-size.ps1

```diff
@@ -0,0 +1,84 @@
+﻿if ( $customSecretsFilePath )
+{
+	$secretsFile = $customSecretsFilePath
+	Write-Host "Using provided secrets file: $($secretsFile | Split-Path -Leaf)"
+}
+if  ($env:Azure_Secrets_File)
+{
+	$secretsFile = $env:Azure_Secrets_File
```

Comment:
I think this file has both tabs and space indentation. Stick to one of them

Reply:
Fixed